hack/local-up-cluster.sh

```diff
@@ -373,19 +385,37 @@ function start_kubelet {
       # unless that file does not already exist; clean up an existing
       # dockerized kubelet that might be running.
       cleanup_dockerized_kubelet
+      cred_bind=""
+      # path to cloud credentails. 
+      cloud_cred=""
+      if [ "${CLOUD_PROVIDER}" == "aws" ]; then
+          cloud_cred="${HOME}/.aws/credentials"
+      fi
+      if [ "${CLOUD_PROVIDER}" == "gce" ]; then
+          cloud_cred="${HOME}/.config/gcloud"
+      fi
+      if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
+          cloud_cred="${CLOUD_CONFIG}"
+      fi
+      if [ "${cloud_cred}" != "" ]; then
+          cred_dir=$(dirname "${cloud_cred}")
```

Comment:
Isn't it sufficient to just mount the required credentials file instead of the entire directory?

Moreover, this seems to mount the entire `$HOME/.config` directory for `gcloud` which is unnecessary (and may be potentially dangerous?).